packages/cli/internal/pkg/cli/slices/slice_utils.go

```diff
@@ -0,0 +1,28 @@
+package slices
```

Comment:
Can you pull this out of the `cli` package? It's not specific to CLI behavior so it can stand alone in its own package at the root of `internal/pkg/`